### PR TITLE
Feature/create process

### PR DESCRIPTION
**Pull request by clarkiness72**

### CODE REVIEW

File: src/components/ProcessForm/ProcessForm.tsx [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+export const ProcessForm: FunctionComponent<IProcessFormProps> = (props) => {
+
+    const [process, setProcess] = useState<IProcess>(getBlankProcess(props.processType));
+    const [validation, setValidation] = useState<IProcessValidation>();
+
+    useEffect(() => {
+        // Update validation whenever a field changes after a submission attempt
+        if (validation) {
+            setValidation(ProcessValidation.validateProcess(process, ["OZI", "OZJ", "OZA"]));
```
Comment (RobertPorterfield) [inline src/components/ProcessForm/ProcessForm.tsx:25]: Is the plan to pull this from a master list somewhere later?

File: src/components/ProcessForm/ProcessForm.tsx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+                    currency = currency.substring(0, commaIndex) + ',' + currency.substring(commaIndex);
+                    commaIndex -= 3;
+                }
+            }
+            setProcess({ ...process, ContractValueDollars: ('$' + currency) });
+        }
+    }
+
+    const submitForm = async () => {
+        const processValidation = ProcessValidation.validateProcess(process, ["OZI", "OZJ", "OZA"]);
```
Comment (RobertPorterfield) [inline src/components/ProcessForm/ProcessForm.tsx:48]: Same as previous comment. Should we update this to be one array passed to both functions? It'll always be the same list, right?

**Review by RobertPorterfield — COMMENTED** [review]
Newly submitted items aren't appearing on the Processes list until a manual page refresh

File: src/components/ProcessForm/ProcessForm.tsx [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+                        <Form.Label className="required"><strong>PoP (months, including options)</strong></Form.Label>
+                        <InfoTooltip id="pop">Agrees with DD2579 Block 8</InfoTooltip>
+                    </Col>
+                    <Col xl="6">
+                        <Form.Control
+                            type="number"
+                            value={process.SboDuration ? process.SboDuration : undefined}
+                            onChange={e => setProcess({ ...process, SboDuration: parseInt(getNumbersOnly(e.target.value)) })}
+                            isInvalid={validation && validation.SboDurationError !== ""}
+                        />
```
Comment (RobertPorterfield) [inline src/components/ProcessForm/ProcessForm.tsx:193]: I believe you can set minimum and maximum value properties (of some sort) on this

File: src/components/ProcessForm/ProcessForm.tsx (hunk 1 above)
Comment (clarkiness72) [reply src/components/ProcessForm/ProcessForm.tsx:25]: Yeah, I made a card for that while I was completing this one

File: src/components/ProcessForm/ProcessForm.tsx (hunk 2 above)
Comment (clarkiness72) [reply src/components/ProcessForm/ProcessForm.tsx:48]: I was leaving any decisions like that for when I implement it as a SP list, which will probably just be implemented as a context provider

File: src/components/ProcessForm/ProcessForm.tsx (hunk 3 above)
Comment (clarkiness72) [reply src/components/ProcessForm/ProcessForm.tsx:193]: Do you know what those limits should be? I can ask Kelly if not

**Review by RobertPorterfield — APPROVED** [review]

File: src/components/ProcessForm/ProcessForm.tsx (hunk 3 above)
Comment (RobertPorterfield) [reply src/components/ProcessForm/ProcessForm.tsx:193]: Based on the email from Kelly I guess it's 1 to infinity

**Pull request merged**